### ISSUE TITLE
Dopamine 2 crashes on startup

### ISSUE THREAD

**Issue by BSoD38** [digimezzo/dopamine-windows#958]
## I'm submitting a...

<pre><code>
[ ] Regression 
[X] Bug report  
[ ] Feature request
</code></pre>

## I'm using these versions

* Windows version: Windows 10 1809
* Dopamine version: 2.0.0.1006

## Reproduction steps

* Seems to happen after restarting Dopamine and/or the computer a number of times. May be related to the track it tries to run at startup.

## Observed behavior

Dopamine starts, show its UI for about half a second, then closes.

## Log file

```
2019-03-10 11:31:21.712|Error|App.xaml.ExecuteEmergencyStop|537|Unhandled Exception. Exception:
System.FormatException: Le format de la chaîne d'entrée est incorrect.
   à System.Number.StringToNumber(String str, NumberStyles options, NumberBuffer& number, NumberFormatInfo info, Boolean parseDecimal)
   à System.Number.ParseUInt32(String value, NumberStyles options, NumberFormatInfo numfmt)
   à System.Convert.ToUInt32(String value)
   à Dopamine.Services.Notification.NotificationService.<PlaybackSuccessSystemNotificationHandler>d__15.MoveNext()
--- Fin de la trace de la pile à partir de l'emplacement précédent au niveau duquel l'exception a été levée ---
   à System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   à System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   à System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)

Stack trace:
   à System.Number.StringToNumber(String str, NumberStyles options, NumberBuffer& number, NumberFormatInfo info, Boolean parseDecimal)
   à System.Number.ParseUInt32(String value, NumberStyles options, NumberFormatInfo numfmt)
   à System.Convert.ToUInt32(String value)
   à Dopamine.Services.Notification.NotificationService.<PlaybackSuccessSystemNotificationHandler>d__15.MoveNext()
--- Fin de la trace de la pile à partir de l'emplacement précédent au niveau duquel l'exception a été levée ---
   à System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   à System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   à System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
```

The french error message reads "The input string's format is incorrect"

**Comment by BSoD38** [comment]
It seems that the error happens when Dopamine tries to read certain tracks. It has happened on both MP3 and FLAC files.

```
2019-03-10 11:51:50.159|Info|PlaybackService.PlayNextAsync|634|Request to play the next track.
2019-03-10 11:51:50.357|Info|PlaybackService.TryPlayAsync|1142|Playing the file E:\Music\375302181_7480ee37dad2adfee934192cf1759260_v3 Turbo Killer.flac. EventMode=False, ExclusiveMode=False, LoopMode=None, Shuffle=True
2019-03-10 11:51:50.359|Error|App.xaml.ExecuteEmergencyStop|537|Unhandled Exception. Exception:
System.FormatException: Le format de la chaîne d'entrée est incorrect.
   à System.Number.StringToNumber(String str, NumberStyles options, NumberBuffer& number, NumberFormatInfo info, Boolean parseDecimal)
   à System.Number.ParseUInt32(String value, NumberStyles options, NumberFormatInfo numfmt)
   à System.Convert.ToUInt32(String value)
   à Dopamine.Services.Notification.NotificationService.<PlaybackSuccessSystemNotificationHandler>d__15.MoveNext()
--- Fin de la trace de la pile à partir de l'emplacement précédent au niveau duquel l'exception a été levée ---
   à System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   à System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   à System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)

Stack trace:
   à System.Number.StringToNumber(String str, NumberStyles options, NumberBuffer& number, NumberFormatInfo info, Boolean parseDecimal)
   à System.Number.ParseUInt32(String value, NumberStyles options, NumberFormatInfo numfmt)
   à System.Convert.ToUInt32(String value)
   à Dopamine.Services.Notification.NotificationService.<PlaybackSuccessSystemNotificationHandler>d__15.MoveNext()
--- Fin de la trace de la pile à partir de l'emplacement précédent au niveau duquel l'exception a été levée ---
   à System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   à System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   à System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)

2019-03-10 11:51:50.359|Info|App.xaml.ExecuteEmergencyStop|540|### FORCED STOP of Dopamine, version 2.0.0.1006 ###
2019-03-10 11:51:50.497|Info|Shell.xaml.Window_Closing|124|### STOPPING Dopamine, version 2.0.0.1006 ###
```

This might be related to the fact that I've enabled system notifications in the settings. Disabling notifications seems to fix the problem, but might need more testing.

**Comment by digimezzo** [comment]
Thanks for logging this issue. Could you add the part of the log which contains that STARTING of Dopamine? It contains some info about your version of Windows 10. Thanks

**Comment by BSoD38** [comment]
Sure. There you go :
```
2019-03-08 18:23:09.430|Info|Initializer.MigrateDatabaseAsync|57|Migrating database
2019-03-08 18:23:09.432|Info|DbMigrator.Migrate|1075|Creating a new database
2019-03-08 18:23:09.521|Info|DbMigrator.CreateDatabase|1146|New database created at C:\Users\Liam\AppData\Roaming\Dopamine\Dopamine.db
2019-03-08 18:23:10.986|Info|App.xaml.InitializeShell|115|### STARTING Dopamine, version 2.0.0.1006, IsPortable = False, Windows version = Microsoft Windows 10 Professionnel (Microsoft Windows NT 10.0.17763.0), IsWindows10 = True ###
2019-03-08 18:23:10.997|Info|App.xaml.InitializeWcfServices|172|CommandService was started successfully
2019-03-08 18:23:10.998|Info|App.xaml.InitializeWcfServices|187|FileService was started successfully
2019-03-08 18:23:11.051|Info|App.xaml.InitializeShell|138|Showing Oobe screen
```

**Comment by digimezzo** [comment]
Fixed in 2921ae055049827233df41e6bf25b6715aa8cac2
